src/components/Button/Button.tsx

```diff
@@ -30,18 +30,17 @@ export class Button extends React.Component<IButtonProps, IButtonState> {
   }
 
   public render(): JSX.Element {
-    let { buttonType, children, icon, description, ariaLabel, ariaDescription, elementType, disabled } = this.props;
+    let { buttonType, children, icon, description, ariaLabel, ariaDescription, elementType} = this.props;
     let { labelId, descriptionId, ariaDescriptionId } = this.state;
 
-    const tag = this.props.elementType === ElementType.button ? 'button' : 'a';
+    const tag = elementType === ElementType.button ? 'button' : 'a';
 
-    const className = css(this.props.className, 'ms-Button', {
+    const className = css((this.props.rootProps && this.props.rootProps.className), 'ms-Button', {
       'ms-Button--primary': buttonType === ButtonType.primary,
       'ms-Button--hero': buttonType === ButtonType.hero,
       'ms-Button--compound': buttonType === ButtonType.compound,
       'ms-Button--command': buttonType === ButtonType.command,
-      'ms-Button--icon': buttonType === ButtonType.icon,
-      'disabled': (elementType === ElementType.anchor && disabled)
```

Comment:
It seems like you're removing a disabled css class here... not sure if intended.